conductor/src/common.hpp

```diff
@@ -0,0 +1,79 @@
+
+#ifndef COCAINE_CONDUCTOR_COMMON_HPP
+#define COCAINE_CONDUCTOR_COMMON_HPP
+
+#include <cocaine/context.hpp>
+#include <cocaine/api/isolate.hpp>
+#include <cocaine/api/service.hpp>
+#include <cocaine/rpc/dispatch.hpp>
+
+#include <cocaine/logging.hpp>
+#include <cocaine/memory.hpp>
+
+#include <cerrno>
+#include <csignal>
+#include <cstdlib>
+#include <cstring>
+#include <system_error>
+#include <atomic>
+
+#include <boost/filesystem/operations.hpp>
+#include <boost/filesystem/path.hpp>
+#include <boost/lexical_cast.hpp>
+
+#include <sys/types.h>
+#include <sys/wait.h>
+#include <unistd.h>
```

Comment:
You don't use theese includes here. So move them out to places where you do use them.